Update documentation instructing the users to use community snippets (#2733)
cursorless-dev/cursorless: packages/cursorless-org-docs/src/docs/user/experimental/snippets.md

![Wrapper snippet demo](images/tryWrapFine.gif)
![Link wrap](images/linkWrap.gif)

## Using community snippets (RECOMMENDED)

> The community Talon files now support their own snippet format. If you'd like to use these snippets for wrapping / cursorless insertion instead of snippets defined in Cursorless, add following line to your `settings.talon` file:

Reviewer: ```suggestion
The community Talon files now support their own snippet format, which is now preferred.

If you'd like to use these snippets for wrapping and inserting with Cursorless instead of the legacy Cursorless snippets, add the following line to your `settings.talon` file:
```
